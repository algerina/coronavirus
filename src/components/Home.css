body {
  text-align: center;
  color: white;
  font-size: x-large;
  padding: 10px;
}

.dark {
  background-color: rgb(72 74 225);
}

.country {
  background-color: rgb(72 74 225);
  padding: 3rem;
  font-size: xx-large;
}

div {
  padding-bottom: 1rem;
}

.back {
  color: white;
  margin: 1rem;
  background-color: rgb(72 74 225);
}
