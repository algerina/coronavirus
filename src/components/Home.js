import React from 'react'

export default function Home() {
  return (
    <div>
      Covid overview
    </div>
  )
}