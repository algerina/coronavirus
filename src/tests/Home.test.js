import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from '../components/Home';

describe('Test for Info', () => {
  const OneCountry = {
    updated: 1642269542433,
    country: 'Afghanistan',
    countryInfo: {
      _id: 4,
      iso2: 'AF',
      iso3: 'AFG',
      lat: 33,
      long: 65,
      flag: 'https://disease.sh/assets/img/flags/af.png',
    },
    cases: 158639,
    todayCases: 0,
    deaths: 7376,
    todayDeaths: 0,
    recovered: 145906,
    todayRecovered: 0,
    active: 5357,
    critical: 1124,
    casesPerOneMillion: 3939,
    deathsPerOneMillion: 183,
    tests: 834349,
    testsPerOneMillion: 20718,
    population: 40272489,
    continent: 'Asia',
    oneCasePerPeople: 254,
    oneDeathPerPeople: 5460,
    oneTestPerPeople: 48,
    activePerOneMillion: 133.02,
    recoveredPerOneMillion: 3622.97,
    criticalPerOneMillion: 27.91,
  };

  test('', () => {
    const tree = renderer.create(
      <Router>
        <Home country={OneCountry} />
      </Router>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
