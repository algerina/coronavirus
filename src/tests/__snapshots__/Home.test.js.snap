// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Info  1`] = `
<div>
  <div
    className="country"
  >
    Afghanistan
  </div>
  <div>
    TOTAL CASES:
     
    158639
  </div>
  <div
    className="dark"
  >
    Total Deaths:
     
    7376
  </div>
  <div>
    Today cases:
     
    0
  </div>
  <div
    className="dark"
  >
    Today Deaths:
     
    0
  </div>
  <div>
    Tests:
     
    834349
  </div>
  <div
    className="dark"
  >
    Recovered:
     
    145906
  </div>
  <div>
    Active cases:
     
    5357
  </div>
  <div
    className="dark"
  >
    Critical cases:
     
    1124
  </div>
  <div>
    <button
      className="back link dim br3 ph3 pv2 mb2 dib white"
      onClick={[Function]}
      type="button"
    >
      Back
    </button>
  </div>
</div>
`;
